.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,23 +10,66 @@ assignees: ''
 
 The release number or commit hash of the version you're using.
 
-## Operating system & version
+## \[Optional\] Setup information
 
-Ex: Windows 10, Ubuntu 18.04, iOS 14.
+If you are reporting a bug about:
+
+- difficulties getting Bevy to build or run on your machine
```

Comment:
How common is this? I see lots of rendering bugs but not so many fails to compile bugs. I have clear biases to what I pay attention to but I haven’t got the impression that building bevy is a common problem and I’d probably order this list roughly according to frequency of occurrence.

Reply:
Frustratingly common. We've had dozens of these, and they suck to reproduce and fix every single time. It's usually some mix of:

- wrong Rust version
- cargo's incremental compile acting up
- conflicting versions of Bevy in dependencies
- missing dependencies, especially on obscure Linux platforms

The rendering issues will at least give some useful diagnostics, although those are usually caused by either low-end hardware or broken support in wgpu / drivers.

See https://github.com/bevyengine/bevy/issues?q=is%3Aopen+is%3Aissue+label%3AC-Startup